modules/olmv1-downloading-bundle-manifests.adoc

```diff
@@ -0,0 +1,150 @@
+[source,terminal]
+----
+$ oc extract registry.redhat.io/openshift-pipelines/pipelines-operator-bundle@sha256:f7b19ce26be742c4aaa458d37bc5ad373b5b29b20aaa7d308349687d3cbd8838
+----
+
+. Change into the `manifests` directory by running the following command:
++
+[source,terminal]
+----
+$ cd manifests
+----
+
+. View the contents of the manifests directory by entering the following command. The output lists the manifests of the resources required to install, manage, and operate your extension.
++
+[source,terminal]
+----
+$ tree
+----
++
+.Example output
+[%collapsible]
+====
+[source,text]
+----
+.
+├── manifests
+│   ├── config-logging_v1_configmap.yaml
+│   ├── openshift-pipelines-operator-monitor_monitoring.coreos.com_v1_servicemonitor.yaml
+│   ├── openshift-pipelines-operator-prometheus-k8s-read-binding_rbac.authorization.k8s.io_v1_rolebinding.yaml
+│   ├── openshift-pipelines-operator-read_rbac.authorization.k8s.io_v1_role.yaml
+│   ├── openshift-pipelines-operator-rh.clusterserviceversion.yaml
+│   ├── operator.tekton.dev_manualapprovalgates.yaml
+│   ├── operator.tekton.dev_openshiftpipelinesascodes.yaml
+│   ├── operator.tekton.dev_tektonaddons.yaml
+│   ├── operator.tekton.dev_tektonchains.yaml
+│   ├── operator.tekton.dev_tektonconfigs.yaml
+│   ├── operator.tekton.dev_tektonhubs.yaml
+│   ├── operator.tekton.dev_tektoninstallersets.yaml
+│   ├── operator.tekton.dev_tektonpipelines.yaml
+│   ├── operator.tekton.dev_tektonresults.yaml
+│   ├── operator.tekton.dev_tektontriggers.yaml
+│   ├── tekton-config-defaults_v1_configmap.yaml
+│   ├── tekton-config-observability_v1_configmap.yaml
+│   ├── tekton-config-read-rolebinding_rbac.authorization.k8s.io_v1_clusterrolebinding.yaml
+│   ├── tekton-config-read-role_rbac.authorization.k8s.io_v1_clusterrole.yaml
+│   ├── tekton-operator-controller-config-leader-election_v1_configmap.yaml
+│   ├── tekton-operator-info_rbac.authorization.k8s.io_v1_rolebinding.yaml
+│   ├── tekton-operator-info_rbac.authorization.k8s.io_v1_role.yaml
+│   ├── tekton-operator-info_v1_configmap.yaml
+│   ├── tekton-operator_v1_service.yaml
+│   ├── tekton-operator-webhook-certs_v1_secret.yaml
+│   ├── tekton-operator-webhook-config-leader-election_v1_configmap.yaml
+│   ├── tekton-operator-webhook_v1_service.yaml
+│   ├── tekton-result-read-rolebinding_rbac.authorization.k8s.io_v1_clusterrolebinding.yaml
+│   └── tekton-result-read-role_rbac.authorization.k8s.io_v1_clusterrole.yaml
+├── metadata
+│   ├── annotations.yaml
+│   └── properties.yaml
+└── root
+    └── buildinfo
+        ├── content_manifests
+        │   └── openshift-pipelines-operator-bundle-container-v1.16.2-3.json
+        └── Dockerfile-openshift-pipelines-pipelines-operator-bundle-container-v1.16.2-3
+----
+====
+
+.Next steps
+
+* View the contents of the cluster service version file using your preferred CLI tool or text editor. Keep this file open as a reference while assigning permissions to the cluster role file in the following procedure.
```

Comment:
should we make it clear that they're looking at the `openshift-pipelines-operator-rh.clusterserviceversion.yaml` and specifically `install.spec.clusterpermissions` ? Though since really the filename doesn't control things maybe we should tell them?:
```bash
$ grep -nr "kind:\ ClusterServiceVersion" *
openshift-pipelines-operator-rh.clusterserviceversion.yaml:2:kind: ClusterServiceVersion
```

Reply:
For this iteration of the docs, I would like to avoid relying on the grep output. But this is good feedback and I will apply it.

Reply:
Addressed: https://github.com/openshift/openshift-docs/pull/87627/commits/cf713d4ece1a2dae5a31b644cfe6246b24d7aaf0